core-web/.husky/pre-commit

```diff
@@ -4,13 +4,42 @@
 
 . "$(dirname "$0")/_/husky.sh"  # Source the husky.sh script
 
+error_occurred=false
+
 # Perform fix operations on the staged files
 perform_fixes() {
-    local files=$1
-    echo "${files}" | xargs -I {} npx nx format:write -- "${root_dir}/{}"  # Format the files
-    echo "${files}" | tr '\n' ',' | sed 's/,$//' | xargs -I {} npx nx affected:lint --files="{}" --fix  # Lint the files and fix any issues
+    local -a files=($1)         # Convert string to array
+    local add_to_index=$2       # Should be 'true' or 'false'
+    local file                  # Declare file variable for use in loop
+
+    if ! npx nx format:write; then
+        error_occurred=true
+    fi
+    if ! npx nx affected --target=lint --exclude=dot-rules,dot-layout-grid,dot-primeng-theme-styles,dot-rules,dotcms,dotcms-field-elements,dotcms-js,dotcms-models,dotcms-webcomponents,dotcdn-e2e,dotcms-block-editor,dotcms-block-editor-e2e,dotcms-ui-e2e,utils-testing,dojo-theme --fix=true; then
+            error_occurred=true
```

Comment:
Why do we to go back to `--excluded` need this if we have the tag `skip:lint`?

Reply:
@fmontes It is now fixed